addons/l10n_id_efaktur/tests/test_l10n_id_efaktur.py

```diff
@@ -82,8 +82,8 @@ def test_efaktur_csv_output_decimal_place(self):
             _csv_row(LT_HEAD_LIST, ','),
             _csv_row(OF_HEAD_LIST, ','),
         )
-        line_4 = '"FK","01","0","0000000000002","5","2019","1/5/2019","12345","l10ntest","","40040","4004","0","","0","44044","0","0","INV/2019/00002 12345","0"\n'
-        line_5 = '"OF","","","100","400.0","40040","0","40040","4004","0","0"\n'
+        line_4 = '"FK","01","0","0000000000002","5","2019","1/5/2019","12345","l10ntest","","40040","4004","0","","0","44044.0","0.0","0","INV/2019/00002 12345","0"\n'
+        line_5 = '"OF","","","100.10","400.0","40040.00","0.00","40040.00","4004.00","0","0"\n'
```

Comment:
```suggestion
        line_5 = '"OF","","","100.00","400.0","40040.00","0.00","40040.00","4004.00","0","0"\n'
```
How did this one change :thinking: